src/legacy/commonUtils.ts

```diff
@@ -59,7 +59,7 @@ export const decryptData = async (ciphertext: string, secretKey: string): Promis
   try {
     return await decryptWithPassword(secretKeyHex, ciphertext)
   } catch (error) {
-    if ((error as Error).message === 'Decryption error') {
```

Comment:
we should wait for pedro to fix this

Reply:
Changed the testing strategy, for now, checking for `CardanoError`, until it gets fixed)